roles/lib_openshift/src/doc/serviceaccount

```diff
@@ -0,0 +1,68 @@
+# flake8: noqa
+# pylint: skip-file
+
+DOCUMENTATION = '''
```

Comment:
Happy to see the docs were updated, and they seem to check out vs the `AnsibleModule` spec. However `./generate.py` must be ran again so that the changes are reflected in the generated `library/` artifacts.

```
[~/rhat/openshift/openshift-ansible/roles/lib_openshift/src] 10:01:43  (oc_serviceaccount)
$ ./generate.py         
[~/rhat/openshift/openshift-ansible/roles/lib_openshift/src] 10:01:44  (oc_serviceaccount⚡)
$ git st                                                   
On branch oc_serviceaccount
Your branch is up-to-date with 'twiest/oc_serviceaccount'.
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

        modified:   ../library/oadm_manage_node.py
        modified:   ../library/oc_serviceaccount.py

no changes added to commit (use "git add" and/or "git commit -a")
```